enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageUpload/ImageUpload.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
```

Comment:
1. This is renamed from `LogoUpload` and made it generic enough so that it works with the favicon upload as well. Previously the preview was hard coded to show the logo for some reason.
2. I've tried to convert this file to TypeScript, but the type is a mess, so I back off.